src/powershell/install-chocolatey.ps1

```diff
@@ -0,0 +1,54 @@
+# Install and configure Chocolatey
+# See: https://community.chocolatey.org/
+
+function Install-Chocolatey {
+    # Download and execute the Chocolatey install script
+    iex ((New-Object System.Net.WebClient).DownloadString("https://community.chocolatey.org/install.ps1"))
+}
+
+function Set-ChocoConfig {
+    # Configure Chocolatey to use Mandiant's VM-Packages repository
+    iex "choco sources add --name vm-packages --source 'https://www.myget.org/F/vm-packages/api/v2;https://myget.org/F/vm-packages/api/v2' --priority 1"
```

Comment:
Please provide some informative commenting around the sources used here.

Reply:
@mcdonnnj resolved with [a526fd9](https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/a526fd9b8491b6ba287a265ec7f67f1a294e58f5)

Reply:
@mcdonnnj resolved with [ea8a0fa](https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/ea8a0faefa9e493d28fc590ead5b8f72ba6fdd8d)